styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
We can lose this file and add `<div class="container-fluid bg-blue-gray fluid-footer-container">...</div>` to `_footer.ejs`